apps/frontpage/lib/get-sitemap-from-tree.ts

```diff
@@ -0,0 +1,31 @@
+import { MetadataRoute } from 'next';
+
+import { generateDocsTree } from './get-tree';
+import { TreeProps, latestVersion } from '@utils';
+
+const docsTree = generateDocsTree();
+
+const startSitemap: MetadataRoute.Sitemap = [];
+
+function getUrlObjectForSlug(slug: string): MetadataRoute.Sitemap[0] {
+  return {
+    url: `https://storybook.js.org${slug.replace(
```

Comment:
We can later optimize with additional properties like `lastModified`, `updateFrequency`, and `priority`.